scripts/local_env_create_vm

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+## Shell Opts ----------------------------------------------------------------
+
+set -o pipefail
+set -xeuo
+
+if [ "$EUID" -ne 0 ]; then
+    echo "Local env needs sudo permissions"
+    exit 1
+fi
+
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+CIFMW_LOCAL_ENV_BASE_DIR="${1}"
+CIFMW_LOCAL_ENV_BACKING_IMG_URL="${2}"
+CIFMW_LOCAL_ENV_PULL_SECRET_PATH="${3}"
+CIFMW_LOCAL_ENV_CALLING_USER=$([ -z "$SUDO_USER" ] && echo "root" || echo "$SUDO_USER")
+CIFMW_LOCAL_ENV_CALLING_USER_HOME=$(eval echo "~$CIFMW_LOCAL_ENV_CALLING_USER")
+
+CIFMW_LOCAL_ENV_IMAGES_DIR="${1}/images"
+CIFMW_LOCAL_ENV_BACKING_IMG="${CIFMW_LOCAL_ENV_IMAGES_DIR}/$(basename ${CIFMW_LOCAL_ENV_BACKING_IMG_URL})"
+CIFMW_LOCAL_ENV_VM_IMG="${CIFMW_LOCAL_ENV_IMAGES_DIR}/ci-cs9.qcow2"
+CIFMW_LOCAL_ENV_VM_NAME="ci-cs9-vm"
+
+if [[ ! -f "$CIFMW_LOCAL_ENV_PULL_SECRET_PATH" ]]; then
+    echo "CIFMW_LOCAL_ENV_PULL_SECRET_PATH must point to an existing pull-secret"
+    exit 1
+fi
+
+if [[ ! -f "$CIFMW_LOCAL_ENV_BACKING_IMG" ]]; then
+    mkdir -p "$CIFMW_LOCAL_ENV_IMAGES_DIR"
+    curl -l -o "$CIFMW_LOCAL_ENV_BACKING_IMG" "${CIFMW_LOCAL_ENV_BACKING_IMG_URL}"
+    curl -l -o "${CIFMW_LOCAL_ENV_BACKING_IMG}.SHA256SUM" "${CIFMW_LOCAL_ENV_BACKING_IMG_URL}.SHA256SUM"
+    (cd "$CIFMW_LOCAL_ENV_IMAGES_DIR" ; sha256sum -c "${CIFMW_LOCAL_ENV_BACKING_IMG}.SHA256SUM")
+fi
+
+rm -rf "$CIFMW_LOCAL_ENV_VM_IMG"
+
+qemu-img create -o backing_file="$CIFMW_LOCAL_ENV_BACKING_IMG",backing_fmt=qcow2 -f qcow2 "$CIFMW_LOCAL_ENV_VM_IMG" "120G"
```

Comment:
On a fresh box, qemu-img, virt-sysprep (coming from guestfs-tools) cli is missing. I need to install these packages separately. 

Can we add the following step at Line 54:
```
sudo dnf -y install qemu-img guestfs-tools
```

Reply:
Good point, I was expecting to have this at some point out of the first iteration, but is doable. I'll add it.

Reply:
The new role/playbook should take care of this. Maybe we'd need to add some more packages in the libvirt_manager role directly.